packages/action-icon-react/src/Icons/IconClear.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export const IconClear = () => (
+    <svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fillRule="evenodd" clipRule="evenodd">
```

Comment:
Maybe my pet peeve, but i think svgs should have a view box, no height, and take width as a argument with 24 as default argument

Reply:
The reasoning is that we can set the size of the icon dynamically to be any size we would like, just by changing a prop